### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#140] [Esercizi - Bertini] [answered]
![Es 7](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/f8a66794-3000-42f6-aa38-a1d32e541cff)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione confermata in classe
- **1)** Se $\mathbb{P}(B^c)=1/4 \Rightarrow \mathbb{P}(B)=1 - 1/4= 3/4$, siccome $\mathbb{P}(A) + \mathbb{P}(B)=1/3+3/4 = 13/12 \> 1$ allora i due eventi devono obbligatoriamente condividere degli eventi elementari per rispettare la condizione $\mathbb{P}(\Omega)=1$. Quindi è impossibile che gli eventi siano disgiunti.

- **2)** Nel caso di eventi disgiunti $\mathbb{P}(A \cup B) = \mathbb{P}(A) + \mathbb{P}(B) \Rightarrow \mathbb{P}(B) = \mathbb{P}(A \cup B) - \mathbb{P}(A) = 3/4 - 1/4 = 2/4$

- **3)** $\mathbb{P}(A \cup B)$ ha valore massimo quando gli eventi sono disgiunti: $\mathbb{P}(A \cup B)= \mathbb{P}(A)+\mathbb{P}(B)=3/8+3/8=6/8$;
mentre ha valore minimo quando gli eventi coincidono: $\mathbb{P}(A \cup B)= \mathbb{P}(A)+\mathbb{P}(B)-\mathbb{P}(A \cap B)=3/8+3/8-3/8=3/8$;
Quindi $3/8 \leq \mathbb{P}(A \cup B) \leq 6/8$, quindi non può valere ne' $1/4$ ne' $7/8$.

- **4)** Dimostriamo il limite inferiore e superiore:
 $\mathbb{P}(A)+\mathbb{P}(B) - \mathbb{P}(A \cap B)= 3/4 + 3/8 -\mathbb{P}(A \cap B) \\leq 1$,
 $\mathbb{P}(A \cap B) \geq 3/4 +3/8 -1 = 9/8-1 = 1/8$ (limite inferiore).
 Siccome $\mathbb{P}(A) \> \mathbb{P}(B)$ è possibile che $B \subset A$, in tal caso $(A \cap B)=B \Rightarrow \mathbb{P}(A \cap B)=3/8$ (limite superiore).

- **5)**  Spostando alcuni membri dell'equazione si ottiene: $1 \geq \mathbb{P}(A) + \mathbb{P}(B) - \mathbb{P}(A \cap B)$
Sappiamo che $\mathbb{P}(A) + \mathbb{P}(B) - \mathbb{P}(A \cap B) = \mathbb{P}(A \cup B)$ per qualsiasi due eventi non necessariamente disgiunti;
Inoltre per la definizione di $\mathbb{P}$ sappiamo che $\mathbb{P}(\Omega) = 1$;
Riscrivendo $\mathbb{P}(\Omega) \geq \mathbb{P}(A \cup B)$, dato che $\mathbb{P}$ è una funzione monotona rispetto all'inclusione di insiemi, la disequazione è rispettata se $(A \cup B) \subset \Omega$ , siccome l'unione di eventi è un operazione chiusa l'evento risultante è per definizione $\subset \Omega$.

**Reply by MostroAle01** [reply]
Non ho capito come hai trovato la P(A ∩ B) nel punto 3, quando l'intersezione non è necessariamente disgiunta.

**Comment by CiottoloMaggico** [comment]
Allego soluzione punto 4:
Per verificare 1/8 <= P(A ∩ B) <= 3/8 possiamo procedere verificando le disuguaglianze separatamente.
P(A∩B) <= 3/8 <=> P(A∩B) <= P(B) => caso peggiore A∩B=B => B⊆A
P(A∩B) >= P(A) + P(B) - 1 (sfruttando la 5) >= 1/8 => P(A)+P(B)-1 = 3/4+3/8-1 = 1/8
La prima e la seconda disuguaglianza possono essere valide contemporaneamente poichè la **5** è rispettata anche per eventi che non sono necessariamente disgiunti.
